include/flatbuffers/flatbuffers.h

```diff
@@ -853,6 +882,16 @@ FLATBUFFERS_FINAL_CLASS
     if (string_pool) delete string_pool;
   }
 
+  /// @brief Get the underlying `vector_downward`
+  /// @return Returns the underlying `vector_downward` buffer
+  vector_downward &GetVectorDownward() { return buf_; }
```

Comment:
This was added primarily as a response to @per-gron concern that protected inheritance can lead to implementation coupling that isn't necessarily obvious from the public interface. I don't have a preference either way, so I'll change this to the original protected access mode change.